universal-application-tool-0.0.1/app/services/export/ExporterService.java

```diff
@@ -206,4 +222,52 @@ private static String pathToHeader(Path path) {
   private static String answerDataKey(AnswerData answerData) {
     return String.format("%s-%d", answerData.blockId(), answerData.questionIndex());
   }
+
+  /**
+   * A string containing the CSV which maps applicants (opaquely) to the programs they applied to.
+   */
+  public String getDemographicsCsv() {
+    return exportCsv(
+        exporterFactory.csvExporter(getDemographicsExporterConfig()),
+        applicantService.getAllApplications());
+  }
+
+  public CsvExportConfig getDemographicsExporterConfig() {
+    ImmutableList.Builder<Column> columnsBuilder = new ImmutableList.Builder<>();
+    // First add the ID, submit time, and submitter email columns.
+    columnsBuilder.add(
+        Column.builder().setHeader("Opaque ID").setColumnType(ColumnType.OPAQUE_ID).build());
+    columnsBuilder.add(
+        Column.builder().setHeader("Submit time").setColumnType(ColumnType.SUBMIT_TIME).build());
+
+    for (QuestionTag tagType :
+        ImmutableList.of(QuestionTag.DEMOGRAPHIC, QuestionTag.DEMOGRAPHIC_PII)) {
+      for (QuestionDefinition questionDefinition :
+          this.questionService.getQuestionsForTag(tagType)) {
+        if (questionDefinition.isEnumerator()) {
+          continue; // Do not include Enumerator answers in CSVs.
```

Comment:
suggestion (if-minor): include enumerators by listing the number of entities as the answer. if you'd rather not right now file an issue

Reply:
Yeah, no idea how, sorry!

Reply:
https://github.com/seattle-uat/civiform/issues/1344